test/powershell/engine/Api/ProxyCommand.Tests.ps1

```diff
@@ -82,7 +82,25 @@ Describe 'ProxyCommand Tests' -Tags "CI" {
         Set-Item -Path function:\TestHelpComment -Value $bodySB
         $newHelpObj = Get-Help TestHelpComment -Full
 
-        $helpObj.Synopsis | Should Be $newHelpObj.Synopsis
+        if($helpObj.Synopsis.Contains("`r`n"))
+        {
+            $helpObjText = ($helpObj.Synopsis).replace("`r`n", [System.Environment]::NewLine).trim()
+        }
+        else
+        {
+            $helpObjText = ($helpObj.Synopsis).replace("`n", [System.Environment]::NewLine).trim()
+        }
+
+        if($newHelpObj.Synopsis.Contains("`r`n"))
```

Comment:
This is almost exactly the same as above, perhaps a helper function instead?

Reply:
Fixed.